en/setup/vscode.md

```diff
@@ -73,6 +75,12 @@ To debug PX4 on SITL:
 
 While debugging you can set breakpoints, step over code, and otherwise develop as normal.
 
+### Hardware Debugging
+
+The instructions in [SWD (JTAG) Hardware Debugging Interface](../debug/swd_debug.html) explain how to connect to the SWD interface on common flight controllers (for example, using the Dronecode or Blackmagic probes).
+
+After connecting to the SWD interface, hardware debugging in VSCode is then typically the same as for [SITL Debugging](#debugging_sitl) except you select the debug target for your firmware - e.g. `px4_fmu-v5`.
```

Comment:
@julianoes @dagar This is a complete guess - is there more to setting up VSCode for hardware debugging than this?

Reply:
@julianoes ?

Reply:
Sorry no idea. I'm ignoring this for now.

Reply:
That's mostly correct, but there are different options per debugger type.

<img width="482" alt="Screen Shot 2020-10-27 at 7 54 56 PM" src="https://user-images.githubusercontent.com/84712/97374269-5119cc80-188e-11eb-85ea-a478a23201fe.png">